### PR TITLE
dev-util/sonarqube-developer

### PR DESCRIPTION
**Pull request by licebmi**
Sonarqube developer server package.

### CODE REVIEW

**Comment by LehningerAdjust** [comment]
@monsieurp Please review.

**Comment by monsieurp** [comment]
CI is broken unfortunately so please ignore for now.

File: dev-util/sonarqube-developer/files/exit-codes.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+--- a/bin/linux-x86-64/sonar.sh
++++ b/bin/linux-x86-64/sonar.sh
+@@ -439,6 +439,7 @@ start() {
+         echo "Started $APP_LONG_NAME."
+     else
+         echo "Failed to start $APP_LONG_NAME."
++	exit 1
```
Comment (monsieurp) [inline dev-util/sonarqube-developer/files/exit-codes.patch:7]: Missing space here:
```suggestion
         exit 1
```

File: dev-util/sonarqube-developer/sonarqube-developer-8.9.8.54436.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	"${FILESDIR}/pidfile.patch"
+	"${FILESDIR}/exit-codes.patch"
+	"${FILESDIR}/configuration.patch"
+)
+
+inherit epatch
+
+src_unpack() {
+	unpack ${P}.zip
+	mv sonarqube-${PV} ${P}
```
Comment (monsieurp) [inline dev-util/sonarqube-developer/sonarqube-developer-8.9.8.54436.ebuild:35]:
```suggestion
        mv sonarqube-${PV} ${P} || die
```

**Review by monsieurp — CHANGES_REQUESTED** [review]

File: dev-util/sonarqube-developer/sonarqube-developer-8.9.8.54436.ebuild (hunk 2 above)
Comment (licebmi) [reply dev-util/sonarqube-developer/sonarqube-developer-8.9.8.54436.ebuild:35]: Is this required?
I mean the zipfile is verified with checksum by portage, and mv is an atomic operation which just migrates an inode. Plus the next step will fail if ${P} doesn't exists.

**Review by monsieurp — APPROVED** [review]

File: dev-util/sonarqube-developer/sonarqube-developer-8.9.8.54436.ebuild (hunk 2 above)
Comment (monsieurp) [reply dev-util/sonarqube-developer/sonarqube-developer-8.9.8.54436.ebuild]: Hey! Yes it's better to add `|| die` statements to make sure the file is indeed renamed.

**Pull request merged**